src/displayapp/screens/SystemInfo.cpp

```diff
@@ -134,7 +137,13 @@ std::unique_ptr<Screen> SystemInfo::CreateScreen2() {
   batteryVoltageBytes[1] = static_cast<uint8_t>(batteryVoltage); // truncate whole numbers
   batteryVoltageBytes[0] =
     static_cast<uint8_t>((batteryVoltage - batteryVoltageBytes[1]) * 100); // remove whole part of flt and shift 2 places over
-  //
+
+  char accel;
+  switch(motionController.DeviceType()) {
+    case Pinetime::Controllers::MotionController::DeviceTypes::BMA421: accel = '1'; break;
+    case Pinetime::Controllers::MotionController::DeviceTypes::BMA425: accel = '5'; break;
+    default: accel = '?'; break;
+  }
```

Comment:
Totally up to you but for cases like these (declare variable then assign variable in each case of a switch statement) I like to use immediately invoked lambdas to directly initialize the variable. That has the added benefit of letting you make the variable const when desired.

Reply:
To add another minor nitpick: this will make an unknown accelerometer display as BMA42?. It really shouldn't happen under normal circumstances and it allows using a char instead of a string, but if we add support for other devices in the future for example it might be incorrect.

Reply:
Right! I moved the conversion to an anonymous function instead. It returns the whole device name instead of only the 1 or 5 :)